app/presenters/search_form_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class SearchFormPresenter
+  attr_accessor :service, :current_user
+
+  def initialize(service, current_user)
+    @service = service
+    @current_user = current_user
+  end
+
+  def result_headers
+    risk_headers = %i[laa_reference firm_name client_name caseworker last_updated risk status_with_assignment]
+    return risk_headers if show_risk_filter?
+
+    %i[laa_reference firm_name client_name caseworker last_updated status_with_assignment]
+  end
+
+  def show_risk_filter?
+    return true if current_user.supervisor? && service == 'analytics'
```

Comment:
My gut instinct is that we don't need the `current_user.supervisor?` check here. We're already ensuring at the controller level that only supervisors can access the analytics service, so this check is redundant, and I think it's sort of out of the responsibility of this presenter to be handling authorisation logic.